### DISCUSSION TITLE
net::ERR_CONNECTION_RESET on all assets when loading github enterprise

### DISCUSSION THREAD

**Discussion by djebita** [community/community#38268] [Enterprise]
Hello,

Following this tutorial : 
https://docs.github.com/en/enterprise-server@3.2/admin/installation/setting-up-a-github-enterprise-server-instance/installing-github-enterprise-server-on-aws

A trial version of github enterprise was installed on AWS but after the instance restart automatically at the end and when i visit the link : ec2-3-125-147-246.eu-central-1.compute.amazonaws.com

I got errors on all static web ressources loaded so no CSS/JS/icons on the web console.

![image](https://user-images.githubusercontent.com/117444272/199949760-20ee69d1-a59c-4138-8600-d859c41e12da.png)

I don't know what was done wrong following the tutorial or what is missing (TLS to activate ? AWS security groups rules to update ? anything else ?)

Any help are very welcome, thanks

Best regards.

**Comment by ossanna16** [comment]
Hi there @djebita and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by djebita** [comment]
Hello,

Disabling subdomain isolation checkbox fixe the issue :
https://docs.github.com/en/enterprise-server@3.4/admin/configuration/configuring-network-settings/enabling-subdomain-isolation

As it's a recommended option to enable, please propose a workaround to enable it and fix also the issue as without it.

Thanks.
